tutormfe/templates/mfe/build/mfe/webpack.prod.tutor.config.js

```diff
@@ -0,0 +1,29 @@
+const { merge } = require('webpack-merge');
```

Comment:
@ghassanmas  This file looks like it has some great optimizations, but they see unrelated to the main purpose of this PR. Would it be OK to break them into separate PRs? Or would that make the image build too big?

Reply:
Yeah make sense to move it to other dedicated PR, because its totally different but a bit related, I initally added it to expirement the concpet, but moving it other PR would make it possible to focus conversation on it. 

The effect of it I think would may be around image of 70~80MB to ~25-30MB, relatively speaking it should be fine withouht given if you take into considratoin that an avarege tutor docker image hundernds of MB, considring, openedx, elasticseach...etc. 

On the other hand, those settings would redcue built time a bit, but again it can move to its dedicated PR/issue.

Reply:
Yup, opened #88 for this